sys-block/hpssacli/hpssacli-4.21.7.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit rpm
+
+MY_PV=$(ver_rs 2 '-')
+
+DESCRIPTION="HPE Smart Storage Administrator (HPE SSA) CLI (HPSSACLI, formerly HPACUCLI)"
+HOMEPAGE="https://support.hpe.com/hpesc/public/swd/detail?swItemId=MTX_8484363847dd4e5ca2970188b7"
+SRC_URI="https://downloads.hpe.com/pub/softlib2/software1/pubsw-linux/p1857046646/v183344/ssacli-${MY_PV}.x86_64.rpm"
+
+LICENSE="hp-proliant-essentials"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+S="${WORKDIR}"
+
+src_unpack() {
+	rpm_src_unpack
+}
+
+src_install() {
+	local -r ssacli_bindir="${WORKDIR}/opt/smartstorageadmin/ssacli/bin"
+
+	dosbin "${ssacli_bindir}"/ssacli
+	dosbin "${ssacli_bindir}"/ssascripting
+	dosbin "${ssacli_bindir}"/rmstr
+
+	dodoc "${ssacli_bindir}/ssacli-${MY_PV}.x86_64.txt"
+
+	gunzip usr/man/man8/ssacli.8.gz || die
```

Comment:
I think we are missing a `BDEPEND` on `app-arch/gzip` for this.

Reply:
[`app-arch/gzip` is part of the `@system` set](https://github.com/gentoo/gentoo/blob/a45623966ad5651c9e3beb9e09428dbc09f1da52/profiles/base/packages#L29), so I think we can omit it.